### ISSUE TITLE
train loss value？

### ISSUE THREAD

**Issue by zhangquanwei962** [Francis-Rings/StableAvatar#20]
I have tried to re-train in the wan2.2 model, the loss value initially is about 0.3, with sampling 1280 samples 81 frames from hallo3 dataset. After 5 epoch, the loss seems to be same. I'm not sure what the converged loss is.. And the inference result is not sync lips.

**Comment by Francis-Rings** [comment]
Hi, for Wan2.1-based StableAvatar, the typical converged loss falls within the range of [0.03, 0.06]. Please note that our training code has only been validated on Wan2.1. The convergence speed depends on the number of trainable parameters, as well as the scale and quality of the video/audio datasets and the learning rate.
